zingolib/src/wallet/disk/testing/tests.rs

```diff
@@ -5,31 +5,20 @@ use zcash_client_backend::ShieldedProtocol;
 use zcash_keys::keys::Era;
 
 use crate::lightclient::LightClient;
+use crate::wallet::disk::testing::examples::{MobileShuffleVersion, VillageTargetVersion};
```

Comment:
should these be on line 18?

Reply:
this is evidence, to me, that this organization is worse.
however, i will merge it for expedience

Reply:
I considered this.   I think a better pattern is to collect the seeds into a `mod` named `seeds`.

I would be in favor of that.  I am not strongly committed to a particular import pattern **except** that I want them to be consistent, and I want code-actions to modify them without my intervention.

Reply:
To me this is evidence that I, a human, am fighting against a tool-or-policy.   Again, I don't care much what the policy for imports is, but it should be consistent, and integrated with code-actions.

If humans are manually formatting import statements then they're wasting their time.